src/components/MyPage/Button.tsx

```diff
@@ -0,0 +1,25 @@
+import RightArrowIcon from "@/assets/icon/ic-right-arrow.svg?react";
+import { layoutStyle, textStyle, versionTextStyle } from "@/components/MyPage/Button.style";
+
+interface buttonProps {
+	icon?: JSX.Element;
+	text: string;
+	arrowIcon: boolean;
+	isVersion: boolean;
+}
+
+const Button = ({ icon, text, arrowIcon, isVersion }: buttonProps) => {
+	return (
+		<div css={layoutStyle}>
+			{icon}
+			<p css={textStyle}>
+				{text}
+				{isVersion && " 1.01.2"}
+				{isVersion && <p css={versionTextStyle}>최신 버전 사용 중</p>}
+			</p>
+			<div>{arrowIcon && <RightArrowIcon />}</div>
```

Comment:
혹시 `{arrowIcon && <RightArrowIcon />}` 여기를 div로 감싼 이유가 있을까요? 
화살표 아이콘이 없는 경우 그냥 빈 div 태그가 남는데, 다른 이유가 있는게 아니라면 불필요한 요소는 안남기는게 좀 더 깔끔하지 않나 싶습니다! 
아래 코드로 변경해서 테스트 해봤는데 UI 상 달라지는 부분은 없더라구요

```suggestion
			{arrowIcon && <RightArrowIcon />}
```